docs/doc/12-unload-data/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Unloading Data from Databend
+slug: ./
+---
+
+Databend provides a variety of tools for loading and unloading data. One of the methods of unloading data from Databend is using the [`COPY INTO <location>`](../14-sql-commands/10-dml/dml-copy-into-location.md) command. 
+
+This command is used to export data from a Databend table or query into a file format such as CSV, Parquet, Or JSON.
+
+
+## Unloading Data
+
+### Creating a Stage
+
+Before unloading data, you need to create a stage, which acts as a storage location for the exported data. 
+You can choose to [create an external or internal stage](../14-sql-commands/00-ddl/40-stage/01-ddl-create-stage.md), depending on your requirements.
```

Comment:
- Is it possible to unload to the user stage, then query from it directly? like "SELECT sum(number) FROM @~ (PATTERN => '.*parquet');"
- I didnt find the document about querying from a stage directly. Can you share the PR of this feature so i can add some document to https://databend-lz99c2p7c-databend.vercel.app/doc/sql-commands/query-syntax/dml-select#from-clause?

Reply:
query from the stage is not ready, we are still working on it.